### ISSUE TITLE
Using hms inside status message leads to error

### ISSUE THREAD

**Issue by cpsievert** [r-lib/cli#269]
Discovered via https://github.com/r-lib/revdepcheck/issues/298

```r
library(cli)
cli_status_update("foo", "bar")
cli_status_clear("foo", result = "done", msg_done = "{hms::as_hms(200)}")
```

```r
#> Error in vec_data(x) * TICS_PER_SECOND : 
#>   non-numeric argument to binary operator
```

**Comment by gaborcsardi** [comment]
I think this has been fixed in dev cli already.

**Comment by cpsievert** [comment]
Can confirm it fixes the original issue (r-lib/revdepcheck#298), thanks!
